scripts/check_docs_uptodate.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Find the directory we exist within
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+cd ${DIR}
+
+tmp=$(mktemp)
+
+echo "checking tools"
+if [ ! -r ../docs/tools.md ]; then
+	echo "docs/tools.md not a readable file?"
+	exit 2
+fi
+./tools-to-doc.sh > $tmp
+diff ../docs/tools.md $tmp
+ret=$?
+[ $ret -gt 0 ] && exit $ret
+
+echo "checking configs"
+if [ ! -r ../docs/config.md ]; then
+	echo "docs/config.md not a readable file?"
+	exit 2
+fi
+./config-to-doc.sh > $tmp
+diff ../docs/config.md $tmp
```

Comment:
looks unfinished

Reply:
what do you mean?

Reply:
it doesn't do anything with the returned value

Reply:
it outputs the diff, if any, and the script exits with the exit code of the diff command, so that we can verify it. but to your point, i will add the rm $tmp which requires adding the explicit exiting with diff's exit code (which currently is not obvious to the reader)